src/utils-lgpl/fcplay.c

```diff
@@ -150,9 +153,8 @@ int main(int argc, char **argv)
 	if (optind >= argc)
 		usage();
 
-	file = argv[optind];
-
-	play_samples(file, card, device, buffer_size, frag, codec_id);
+	play_samples(argv, card, device, buffer_size, frag,
+					codec_id, optind, argc - optind, gapless);
```

Comment:
this does not look right to me, we should be sending gapless variable here

Reply:
thanks for review,

ok, will sent the gapless variable along with total file count variable.
In above function argc - optind will give total file count, for next post will send file count variable.